mage/main.wasp

```diff
@@ -1,6 +1,6 @@
 app waspAi {
   wasp: {
-    version: "^0.18.0"
+    version: "^0.19.0"
```

Comment:
Note that this you don't really need to bump with the rest. Mage is not an example app, nor is it used for testing. We can bump it at its own tempo, when we feel like updating it.